Examples/d/extend/example.h

```diff
@@ -13,15 +13,15 @@ class Employee {
 	Employee(const char* n): name(n) {}
 	virtual std::string getTitle() { return getPosition() + " " + getName(); }
 	virtual std::string getName() { return name; }
-	virtual std::string getPosition() const { return "Employee"; }
+	virtual std::string getPosition() { return "Employee"; }
```

Comment:
Why are you removing `const` from this example?

Reply:
I know that using `const` does make sense.
But test breaks with it.
Do not have time to fix all.
We can add a 'FIXME' comment, if you please.